package.json

```diff
@@ -40,5 +40,8 @@
 		"node_modules/",
 		"test/"
 	],
+	"prettierIgnore": [
+		"test/"
+	],
```

Comment:
I'd like all files to be formatted, what's the reason for ignoring the `test` directory?

Also, this `package.json` key unfortunately isn't supported yet: https://github.com/prettier/prettier/issues/3460

Reply:
This actually doesn't work, I just need to remove it.